modules/guides/pages/serverless/lambda.adoc

```diff
@@ -1,30 +1,23 @@
 = Lambda
 :description: Deploying as an AWS Lambda function
 
-The `benthos-lambda` distribution is a version of {page-component-title} specifically tailored
+You can use either of the following distributions to deploy {page-component-title} as an AWS Lambda function:
+
+* The `redpanda-connect-lambda` distribution is specifically tailored
 for deployment as an AWS Lambda function on the `go1.x` runtime,
 which runs Amazon Linux on the `x86_64` architecture.
-The `benthos-lambda-al2` distribution supports the `provided.al2` runtime,
+
+* The `redpanda-connect-lambda-al2` distribution supports the `provided.al2` runtime,
```

Comment:
As discussed, I've assumed that a new distro will be available with the name change, but let me know if that isn't going to be possible.

Reply:
Yep, just getting the changes ready on a PR now so it'll be ready for the next release. I think it's fine to deploy these docs updates before then as it's a lot less confusing than the current docs.